compiler/circle-mpqsolver/src/MPQSolver.cpp

```diff
@@ -34,6 +39,55 @@ void MPQSolver::set_save_intermediate(const std::string &save_path)
   _hooks = std::make_unique<core::DumpingHooks>(save_path);
 }
 
+void MPQSolver::set_mpq_options(MPQOptions &options) { _options = options; }
+
+LayerParams MPQSolver::get_frozen_params() const
+{
+  LayerParams params;
+  for (auto node_to_param : _frozen._node_to_param)
+  {
+    params.push_back(std::make_shared<LayerParam>(node_to_param.second));
+  }
+
+  return params;
+}
+
+void MPQSolver::resolve_patterns(luci::Module *module)
+{
+  _frozen._node_to_param.clear();
+
+  for (auto pattern : _options._patterns)
+  {
+    std::unique_ptr<pattern::PatternResolver> resolver;
+    switch (pattern)
+    {
+      case QuantizationPattern::Q8LayerNormWithQ16Variance:
+      {
+        resolver = std::make_unique<pattern::Q8LayerNormWithQ16VarianceResolver>();
+      }
+      break;
```

Comment:
is there any reason not to use like this?
```suggestion
        resolver = std::make_unique<pattern::Q8LayerNormWithQ16VarianceResolver>();
        break;
```

Reply:
> is there any reason not to use like this?

No. For now there is no reason to use `{}`.